tests/e2e/lifecycle/mcg/test_mcg_namespace_lifecyle.py

```diff
@@ -108,8 +108,11 @@ def test_mcg_namespace_lifecycle(
         logger.info(f"Noobaa account: {user.email_id} with S3 access created")
 
         bucket_policy_generated = gen_bucket_policy(
-            user_list=user.email_id,
-            actions_list=["PutObject", "GetObject"],
+            user_list=[user.email_id],
+            actions_list=["PutObject", "GetObject"]
+            if float(config.ENV_DATA["ocs_version"]) <= 4.6
+            else ["DeleteObject"],
+            effect="Allow" if float(config.ENV_DATA["ocs_version"]) <= 4.6 else "Deny",
```

Comment:
We should move these if/else statements ouside of the `get_bucket_policy` function call and assign the values to variables which we can use when calling the function.

Reply:
ack